tutorials/3-run-node/index.md

```diff
@@ -449,17 +755,192 @@ tx: null
 txhash: D2CCFD91452F8C144BB1E7B54B9723EE3ED85925EE2C8AD843392721D072B895
 ```
 
-The command output includes useful information, such as `gas_used`.
+The command output could include useful information, such as `gas_used`. However, here it did not have time to collect the information because the command returned before the transaction was included in a block.
+
+You can call back the transaction information using the transaction hash above:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
+
+```sh
+$ ./build/simd query tx D2CCFD91452F8C144BB1E7B54B9723EE3ED85925EE2C8AD843392721D072B895
```

Comment:
Do we want to make it clearer here and use something like <txhash>? Not sure if its necessary

Reply:
Ha! I had to edit your comment to see the hidden HTML tag :)
Will do.